### PR TITLE
[FIX JENKINS-39587] refactor to not use context but use standard properties

### PR DESCRIPTION
**Pull request by scherler**
# Description

In some circumstances, that are not reproducible, the listing of pipelines has shown a blank list.
I do not why I had this circumstances, but the usage of context had been wrong anyhow in this place.

https://issues.jenkins-ci.org/browse/JENKINS-39587

# Submitter checklist
- [ ] Link to JIRA ticket in description, if appropriate.
- [ ] Change is code complete and matches issue description
- [ ] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.
- [ ] Ran Acceptance Test Harness against PR changes.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

@jenkinsci/code-reviewers @reviewbybees

### CODE REVIEW

**Comment by alecharp** [comment]
:bee: once the build is validated

**Comment by alecharp** [comment]
sorry but :bug: for the moment. see http://pr.sandbox.blueocean.io/pr-590/blue/organizations/jenkins/multibranch/activity/ which is not working and goes into an infinite loop

File: blueocean-dashboard/src/main/js/OrganizationPipelines.jsx [hunk 1]
```diff
@@ -132,10 +121,17 @@ class OrganizationPipelines extends Component {
...
+        let pipelines = null;
+        if (!allPipelines && !organizationPipelines) {
             return null;
         }
-        return this.props.children;
+        if (allPipelines && allPipelines.$success) {
+            pipelines = allPipelines;
+        } else if (organizationPipelines && organizationPipelines.$success) {
+            pipelines = organizationPipelines;
+        }
```
Comment (tfennelly) [inline blueocean-dashboard/src/main/js/OrganizationPipelines.jsx]: So reading the above code .... both `allPipelines` and `organizationPipelines` cannot both be non-null at the same time, or does `allPipelines` always take precedence when they're both non-null?

Comment (tfennelly) [inline blueocean-dashboard/src/main/js/OrganizationPipelines.jsx]: Maybe a comment here for other readers would be a good idea.

**Comment by scherler** [comment]
@alecharp that is fixed now

**Comment by scherler** [comment]
@alecharp that is fixed now

**Comment by alecharp** [comment]
:bee: nicely done!

**Comment by i386** [comment]
@scherler good to see a bug fix like this get done 💯

**Pull request merged**